keyboards/jones/v1/keymaps/via/rules.mk

```diff
@@ -0,0 +1,5 @@
+# Use VIA
+VIA_ENABLE = yes
+
+# Use I2C connected External EEPROM
+EEPROM_DRIVER = i2c
```

Comment:
is there a reason that this is enabled just for via, and not globally for the keyboard?

Reply:
I added External EEPROM, just to increase layer count on VIA environment.

Without VIA, we have enough EEPROM to use.
And also I2C may increase firmware size.

So, it's just for VIA.

Reply:
If the hardware is physically present, then it should be enabled on the keyboard level, IMO. 

Namely, there are other things that could use the EEPROM.  Like QMK XAP.

Could you please move all of the eeprom config to the keyboard level config files?

Reply:
I understand what you mean, something big will come.
(Anyway, I just heard QMK XAP last night.)

I moved EEPROM configs to keyboard level, except DYNAMIC_KEYMAP_LAYER_COUNT.
I think it is keymap dependent config.
Am I right?

Reply:
Awesome! 

And I'd prefer that the dynamic keymap define be in the keyboard config, as there are people that like to customize their board and still have the via functionality.  Also, while the dynamic keymap stuff is used by VIA, it's not strictly limited to VIA.

Reply:
Thanks to inform me that.
I moved the rest of DYNAMIC_KEYMAP config to keyboard level.

Reply:
you are very welcome!  And thanks!